public/javascripts/default_scripts.js

```diff
@@ -88,18 +88,19 @@ function SensorDropdown(sensor) {
               `data-bs-placement="right" title="process time: &nbsp; ${doc.rate.toPrecision(3)} ms  \n`+
               `last cycle: &nbsp; ${((now-doc.heartbeat)/1000 || 0).toPrecision(1)} s \n`+
               `last error: &nbsp; ${doc.cycles - doc.error} cycles ago"><span class="visually-hidden">X</span></span></td>`;
-          let goto_btn = `<button class="btn btn-primary action_button" `+
-              `onclick="location.href='pipeline?pipeline_id=${pl_name}'"> Go to</button>`;
+          let stop_btn = `<button class="btn btn-danger action_button" `+
+              `onclick="SendToHypervisor('pl_${flavor}', 'pipelinectl_stop ${pl_name}')">`+
+              `<i class="fas fa-solid fa-stop"></i>Stop</button>`;
+          let start_btn = `<button class="btn btn-success action_button" onclick="SendToHypervisor('pl_${flavor}', 'pipelinectl_start ${pl_name}')"><i class="fas fa-solid fa-play"></i>Start</button>`;
+          let restart_btn = `<button class="btn btn-primary action_button" onclick="SendToHypervisor('pl_${flavor}', 'pipelinectl_restart ${pl_name}')"><i class="fas fa-solid fa-rotate"></i>Restart</button>`;
+          let silence_btn = `<button class="btn btn-secondary action_button" onclick="SendToHypervisor('pl_${flavor}', 'pipelinectl_silent ${pl_name}')"><i class="fas fa-solid fa-bell-slash"></i>Silence</button>`;
```

Comment:
Does this work? Don't you need to give the time to slice for?

Reply:
This part got out of my mind, it was just silencing until the end of time.